scripts/sysman/PROG.rst

```diff
@@ -1974,6 +1973,21 @@ in the structure ${s}_ras_state_t. The table below describes the categories:
 |                                                                   | that have occurred in                                        | that have occurred in                                          |
 |                                                                   | the display.                                                 | the display.                                                   |
 +-------------------------------------------------------------------+--------------------------------------------------------------+----------------------------------------------------------------+
+| ${S}_RAS_ERROR_CAT_MEMORY_ERRORS      | Number of ECC                                                | Number of ECC                                                  |
+|                                                                   | correctable errors                                           | uncorrectable errors                                           |
+|                                                                   | that have occurred in                                        | that have occurred in                                          |
+|                                                                   | memory - GDDR/HBM.                                           | memory - GDDR/HBM                                              |
++-------------------------------------------------------------------+--------------------------------------------------------------+----------------------------------------------------------------+
+| ${S}_RAS_ERROR_CAT_SCALE_ERRORS       | Number of                                                    | Number of ECC                                                  |
+|                                                                   | correctable errors                                           | uncorrectable errors                                           |
+|                                                                   | that have occurred in                                        | that have occurred in                                          |
+|                                                                   | scale fabric.                                                | scale fabric.                                                  |
++-------------------------------------------------------------------+--------------------------------------------------------------+----------------------------------------------------------------+
+| ${S}_RAS_ERROR_CAT_L3FABRIC_ERRORS    | Number of ECC                                                | Number of ECC                                                  |
```

Comment:
"Number of ECC " to number of L3 Fabric errors ?

Reply:
same as above